From 696206a798647adc73faab26ab5b7543df939372 Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Mon, 17 Jul 2023 17:54:59 +0200
Subject: [PATCH] ci: drop super-linter's shellcheck

It's been a while since we introduced Differential ShellCheck and it
proved to be quite useful (and in some ways even better than the shellcheck
run by super-linter). So, to have only one linter scream at us for not
knowing how to write bash properly, let's drop the super-linter's one in
favor of Differential ShellCheck.

Follow-up for https://github.com/systemd/systemd/pull/24328#pullrequestreview-1074127504

(cherry picked from commit c4b167f857fe5c228da4d4950a3eae13f53c645c)

Related: RHEL-27512
---
 .github/workflows/linter.yml | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/.github/workflows/linter.yml b/.github/workflows/linter.yml
index eddd350122..8dd3b075ad 100644
--- a/.github/workflows/linter.yml
+++ b/.github/workflows/linter.yml
@@ -32,13 +32,5 @@ jobs:
         uses: github/super-linter/slim@01d3218744765b55c3b5ffbb27e50961e50c33c5
         env:
           DEFAULT_BRANCH: main
-          MULTI_STATUS: false
-          # Excludes:
-          #   - man/.* - all snippets in man pages (false positives due to
-          #              missing shebangs)
-          #   - .*\.(in|SKELETON) - all template/skeleton files
-          #                         except kernel-install
-          FILTER_REGEX_EXCLUDE: .*/(man/.*|([^k]|k(k|ek)*([^ek]|e[^kr]))*(k(k|ek)*e?)?\.(in|SKELETON))$
           VALIDATE_ALL_CODEBASE: false
-          VALIDATE_BASH: true
           VALIDATE_GITHUB_ACTIONS: true
